OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -891,21 +891,22 @@ private void LoadInterface(InterfaceDescriptor descriptor) {
     }
 
     private void LoadDescriptor(ElementDescriptor descriptor) {
+        // Note: Id has a getter to ensure Id.Value is never null
```

Comment:
I don't see why `Value` is marked nullable in the first place. The value isn't nullable in the constructor, and nothing ever gives it `null`.